src/status_im/chat/models/message.cljs

```diff
@@ -72,10 +74,15 @@
   (assoc message :outgoing (= from (:current-public-key db))))
 
 (fx/defn add-message
-  [{:keys [db] :as cofx} batch? {:keys [chat-id message-id clock-value] :as message} current-chat?]
+  [{:keys [db] :as cofx} batch? {:keys [chat-id message-id clock-value timestamp content from] :as message} current-chat?]
   (let [prepared-message (-> message
                              (prepare-message chat-id current-chat?)
                              (add-outgoing-status cofx))]
+    (when (and platform/desktop?
+               (not= from (:current-public-key db))
+               (get-in db [:account/account :desktop-notifications?])
+               (< (time/seconds-ago (time/to-date timestamp)) 86400))
+      (#(.sendNotification react/desktop-notification (:text content))))
```

Comment:
```
(.sendNotification react/desktop-notification (:text content))
```
should be enough instead of
```
(#(.sendNotification react/desktop-notification (:text content)))
```